database/build.go

```diff
@@ -45,6 +45,7 @@ type Build struct {
 	Started       sql.NullInt64      `sql:"started"`
 	Finished      sql.NullInt64      `sql:"finished"`
 	Deploy        sql.NullString     `sql:"deploy"`
+	DeploymentID  sql.NullInt64      `sql:"deployment_id"`
```

Comment:
```suggestion
	DeployID  sql.NullInt64      `sql:"deployment_id"`
```
Thoughts on this for consistency with `Deploy` and `DeployPayload`?